app/routes/beelievers-auction.tsx

```diff
@@ -9,9 +9,9 @@ export async function loader() {
 
 export async function action() {
 	// Using an empty string as a placeholder for eligibility checks.
-	const isEligible = await checkEligibility("");
+	const eligible = await checkEligibility("");
```

Comment:
here we need to send Sui wallet address, and has to be reloaded on connect / reconncet (if user changes the wallet or loads other address)